subsystems/TypeHandlerLibrary/build.gradle.kts

```diff
@@ -6,7 +6,10 @@ plugins {
     `java-library`
 }
 
-apply(from = "$rootDir/config/gradle/common.gradle")
+apply(from = "$rootDir/config/gradle/publish.gradle")
+
+group = "org.terasology"
+version = project(":engine").version
```

Comment:
Ah there we go, there's the version link, which is fine if they'll always be in sync, which seems reasonable :-)